symfony/framework-bundle/3.3/public/index.php

```diff
@@ -21,7 +21,9 @@
     Debug::enable();
 }
 
-// Request::setTrustedProxies(['0.0.0.0/0'], Request::HEADER_FORWARDED);
+if (($trustedProxies = getenv('TRUSTED_PROXIES')) && ($trustedHeaders = getenv('TRUSTED_HEADERS'))) {
```

Comment:
It makes it easier to read when the first and second part use the same amount of parentheses. There is no drawback anyway.